nums/numpy/api/creation.py

```diff
@@ -1116,3 +1116,74 @@ def zeros_like(prototype, dtype=None, order="K", shape=None):
     if order is not None and order != "K":
         raise NotImplementedError("Only order='K' is supported.")
     return zeros(shape, dtype)
+
+
+def asarray(a, dtype=None, order=None, *, like=None):
+    """Convert the input to an array.
+
+    Parameters
+    ----------
+    a : array_like
+        Input data, in any form that can be converted to an array.  This
+        includes lists, lists of tuples, tuples, tuples of tuples, tuples
+        of lists and ndarrays.
+    dtype : data-type, optional
+        By default, the data-type is inferred from the input data.
+
+    Returns
+    -------
+    out : ndarray
```

Comment:
Change this to `BlockArray` and any other mentions of `ndarray` to `BlockArray`

Reply:
Yeah. Was talking to @elibol about potentially renaming `BlockArray` to `ndarray`, but you're right. Shouldn't prematurely change the name.

Reply:
Fixed